pages/users/signUp/index.tsx

```diff
@@ -36,6 +37,7 @@ const UserSignUpForm1Schema = Joi.object<UserSignUpForm1Values>({
     .empty("")
     .allow(null),
   password: Joi.forbidden().required(),
+  adminNotes: Joi.string().trim().required(),
```

Comment:
We actually don't want adminNotes to be required because it won't allow the user to move onto the next page